x/evm/types/keys.go

```diff
@@ -15,8 +17,19 @@ const (
 
 	// MemStoreKey defines the in-memory store key
 	MemStoreKey = "mem_evm"
+
+	UserSmartContractStoreKeyPrefix = "user-smart-contract-"
+	UserSmartContractKeyPrefix      = "contract-"
```

Comment:
I don't think we're using trailing hyphens anywhere else for prefixes. What's your reasoning behind that?

Reply:
I was building the keys manually with the prefix instead of using a proper prefix store. I'll look into changing it.